low-level/framework/messaging/ingress_processor.py

```diff
@@ -248,18 +261,17 @@ def _send_to_msg_handler(self, msgType, message, uuid):
             self._write_internal_msgQ(EgressProcessor.name(),
                                       ack_msg)
 
-    def _read_config(self):
+    def _init_config(self):
         """Read config for messaging bus."""
         # Make methods locally available
-        self._node_id = Conf.get(SSPL_CONF,
-                                 f"{CLUSTER}>{SRVNODE}>{self.NODE_ID_KEY}",
-                                 'SN01')
+        self._node_id = Conf.get(GLOBAL_CONF, NODE_ID_KEY, 'SN01')
+        self._consumer_group_prefix = Conf.get(
+            SSPL_CONF, f"{self.PROCESSOR}>{self.CONSUMER_GROUP}",
+            'cortx_monitor')
+        self._consumer_group = self._consumer_group_prefix + str(self._node_id)
```

Comment:
here we can append with - and then nodeid

Reply:
Done